### ISSUE TITLE
`boxplot` doesn't have `alpha` keyword

### ISSUE THREAD

**Issue by lassepe** [JuliaPlots/Makie.jl#3099]
First of all, thank you for landing support for the `alpha` keyword in Makie! :tada: I am enjoying that feature quite a lot!

I found what seems to be an edge case with that feature: `Makie.boxplot` seems to ignore this keyword (at least in GLMakie).

```julia
using GLMakie

data = rand(100)
boxplot(ones(length(data)), data; color = :black, alpha = 0.5)
```
![image](https://github.com/MakieOrg/Makie.jl/assets/10076790/bb32c981-737e-4eac-875a-7f42e19cf213)

Expected result:
```julia
boxplot(ones(length(data)), data; color = (:black,0.5))
```

![image](https://github.com/MakieOrg/Makie.jl/assets/10076790/43c0b070-0f5e-49d7-adf5-1d853e31fb77)

**Comment by jkrumbiegel** [comment]
yeah so the problem is that alpha is currently only implemented for the primitives, we still need some time to add it to all the non-primitive recipes (which should just forward it in this case)

**Comment by ffreyer** [comment]
This has been fixed
